src/app/pages/RuntimeTransactionDetailPage/index.tsx

```diff
@@ -364,6 +364,15 @@ export const RuntimeTransactionDetailView: FC<{
             </>
           )}
 
+          {!transaction.encryption_envelope && transaction.body?.init_code && (
```

Comment:
Can we end up with two `transaction.rawData` rows or it's impossible to have `!!transaction.body?.data` and `!transaction.encryption_envelope && transaction.body?.init_code` in one tx ?

Reply:
Great question. I fwd-ed it to the Nexus guys for clarification.